packages/vhd-lib/src/createReadableSparseStream.js

```diff
@@ -62,7 +61,7 @@ function createBAT(
  *  "fragment" designate a chunk of incoming data (ie probably a VMDK grain), and "block" is a VHD block.
  * @param diskSize
  * @param fragmentSize
- * @param fragmentLogicalAddressList
+ * @param fragmentLogicalAddressList an iterable returning LBAs in multiple of fragmentSize
```

Comment:
Should it be in a comment?

Reply:
isn't a doc block a comment?

Reply:
Ah yes, I missed that all that stuff was a comment block! Sorry for the noise :)